discovery/v2.ts

```diff
@@ -5412,6 +5399,15 @@ namespace DiscoveryV2 {
     notices?: Notice[];
   }
 
+  /** Result for the `pair` aggregation. */
+  export interface QueryPairAggregationResult {
```

Comment:
weird, I don't see this being used anywhere

Reply:
It is used in this interface: `QueryAggregationQueryPairAggregation`